proposals/drafts/campaign_finance_filings.rst

```diff
@@ -0,0 +1,294 @@
+    they're involved. It will mostly be people but not always, and as lovely as
+    it would be to have some way to disambiguate the two I'm not optimistic that
+    will be possible anytime soon.
+
+Section
+    A container for a unit of meaning inside a Filing, that isn't inherent to
+    the basic process of filing a document with an agency.
+
+Rationale
+=========
+
+Political committees file statements with their regulators in order to disclose
+funders, expenditures, organizational ties and other financial details.
+Filings encapsulate some number of possible specific disclosures, and some
+filings can contain more than one type of disclosure, so this proposed option is
+to consider each actual filing as an object and then each specific disclosed
+piece of information as associated with that object. So a CommitteeNameChange
+would be associated with a Filing, along with a CommitteeAddressChange and a
+CommitteeOfficerAddition. A series of CommitteeContributions and
+CommitteeExpenses would be attached to one Filing. This way we don't need to
+model the specific disclosure forms in each state, and just directly extract
+the most meaningful parts of the filing.
+
+We want to make it easy for people to do manipulations on top of this data,
+while ensuring that this system itself does as little manipulation as possible
+to the underlying data from reports filed. For instance, many records include
+Person entities - these are deterministically extracted from just the available
+filing data, e.g. an address, employer, occupation and name are all extracted
+for each contribution. Deduplication can then happen with each of these
+entities, but they are themselves totally reproducible from just the filing
+information.
+
+This model entails that Filings have Sections - a Filing can, and often will,
+have multiple Sections. As much of the specific logic as possible should be
+pushed down into Sections, which offer flexibility by allowing us to compose
+them together to reproduce the data from state-specific reports. We can model
+a particular state filing form as a Filing with a specific set of expected
+Sections to be present. Some attributes that might make sense as properties of a
+Filing, such as Amendments, are here presented as Sections; the line between the
+two is muddy, but this proposal tries to keep Filings as minimal as possible.
+
+Finally, the different entity types presented here present either current or
+likely future conflicts with other entities in the Open Civic Data namespace, so
+this proposal doesn't make any attempt to resolve those conflicts since they
+seem ancillary for now.
+
+TODO
+----
+* Make filing activity actions use same pattern as OCD bill actions (@aepton)
+
+Questions to answer
+-------------------
+* How to model Committee Status and Committee Purpose inside of OCD Organization
+  models?
+
+Implementation
+==============
+
+Filing
```

Comment:
Ok, so it sounds like we should have a repeated, optional field indicating which election(s) this filing applies to, and since we're also introducing the notion of an Election object [in a separate PR](https://github.com/opencivicdata/docs.opencivicdata.org/pull/62), this seems straightforward.

@LindsayYoung does that capture what you were looking for in terms of flexibility? I'd absolutely love to be able to use this to model federal elections/data.

@jsfenfen I think what you're describing is what I have in mind for how this system would work for an end user - we do the work of translating federal/state/local filings into this standard scheme, and each state/federal/local parser is responsible for doing The Right Thing for that jurisdiction, such that it's easy for people to do cross-jurisdictional comparisons.

For an end user, there should be some interface to this data, and since the jurisdictional parsers have done the heavy lifting of saying that "F3P gets processed this way, and F3X gets processed that way" then the end-user-interface system can make the comparisons users have in mind in a straightforward way.